test/software/amazon/ion/TimestampTest.java

```diff
@@ -860,6 +860,57 @@ public void testNewTimestampFromLong()
         assertEquals("2010-02-01T10:11:12.340Z", ts.toZString());
     }
 
+    /**
+     * Regression test for https://github.com/amzn/ion-java/issues/160
+     */
+    @Test
+    public void testNewTimestampFromLongYearOneRegressionBug()
```

Comment:
"FromLongYearOneRegressionBug" ?  perhaps just "FromYearOneRegressionBug"?

Reply:
Yes.  Not sure where "Long" came from.